flink-ml-lib/src/main/java/org/apache/flink/ml/classification/logisticregression/LogisticRegressionModel.java

```diff
@@ -149,22 +152,14 @@ public Row map(Row dataPoint) {
                                 getRuntimeContext().getBroadcastVariable(broadcastModelKey).get(0);
                 coefficient = modelData.coefficient;
             }
-            DenseVector features = ((Vector) dataPoint.getField(featuresCol)).toDense();
-            Row predictionResult = predictOneDataPoint(features, coefficient);
-            return Row.join(dataPoint, predictionResult);
-        }
-    }
+            Vector features = (Vector) dataPoint.getField(featuresCol);
+
+            LogisticRegressionModelServable servable =
+                    new LogisticRegressionModelServable(
```

Comment:
1. Without the `LogisticRegressionModelServable(LogisticRegressionModelData)` constructor, every time we want to construct the `LogisticRegressionModelServable` with an existing `LogisticRegressionModelData`, we need to serialize the instance of `LogisticRegressionModelData` to bytes and then deserialize the bytes via `setModelData(InputStream... modelDataInputs)`, it may be expensive if the model data is large. I prefer to keep the constructor but make it `protected` or `default`.
2.   Yes we should forward parameters from model to servable, but the `LogisticRegressionModelServable` doesn't need any parameters from model to do predictions. If we are supporting `MinMaxScalerModelServable`, developers can pass the params by `servable.setMin()` directly.

Reply:
Making `LogisticRegressionModelServable(LogisticRegressionModelData) ` protected sounds good to me.

However, asking developers to identify and forward the selectively parameters from model to servable is not very convenient and introduces chances of human error. It will much more convenient for Flink ML to provide infra that algorithm developers can use to forward parameters, similar to how the existing Estimators forward parameters to Model using `ParamUtils.updateExistingParams`.